### PR TITLE
chore: Update Dev docs

### PR DESCRIPTION
**Pull request by mhomolak**

### CODE REVIEW

**Comment by parkerpearson** [comment]
Feel free to merge this whenever you're ready.

File: component-creation-guide.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Let everyone know your idea for a component and begin to gather feedback. This i
 Describe and specify your component, from general features to usage guidelines to specific colors and dimensions.
 
 3. **Implement**  
-Put your design to code and produce the final package which can be consumed by other teams.
+Transform your design into code and produce the final package which can be consumed by other teams.
```
Comment (StommePoes) [inline component-creation-guide.md]: oooooh, sounds like an ad for something!

File: membership-spec/index.md [hunk 2]
```diff
@@ -210,7 +210,7 @@ dev_reqs:
         extras:
           - name: Explanation
             content: |
-              The starter kit implements best practices like testing, linting, bundling, transpiling ES6 to ES5, etc. It codifies a long list of decisions that you no longer have to make to get rolling. It saves you from the long, painful process of wiring it all together into an automated development environment and build process.
+              The archetype implements best practices like testing, linting, bundling, transpiling ES6 to ES5, etc. It codifies a long list of decisions that you no longer have to make to get rolling saving you from wiring it all together into an automated development environment and build process.
```
Comment (StommePoes) [inline membership-spec/index.md]: This sentence got long:
It codifies a long list of decisions that you no longer have to make to get rolling saving you from wiring it all together into an automated development environment and build process.

We at least need a comma after rolling.

File: membership-spec/index.md [hunk 3]
```diff
@@ -210,7 +210,7 @@ dev_reqs:
         extras:
           - name: Explanation
             content: |
-              The archetype implements best practices like testing, linting, bundling, transpiling ES6 to ES5, etc. It codifies a long list of decisions that you no longer have to make to get rolling saving you from wiring it all together into an automated development environment and build process.
+              The archetype implements best practices like testing, linting, bundling, transpiling ES6 to ES5, etc. It codifies a long list of decisions that you no longer have to make, saving you from wiring it all together into an automated development environment and build process.
```
Comment (StommePoes) [inline membership-spec/index.md]: ++

**Pull request merged**